introduction.md

```diff
@@ -0,0 +1 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut quis lacus turpis. Fusce varius, dui eu dapibus hendrerit, eros erat sollicitudin nisi, sed tempus eros massa sed tortor. Donec volutpat urna id porta dapibus. Pellentesque et elementum lorem, nec auctor diam. Vivamus accumsan metus urna, vel lacinia felis rutrum vitae. Nulla ullamcorper imperdiet sem quis vestibulum. Nullam cursus lorem in ligula aliquam, sed malesuada sapien ullamcorper. Vestibulum aliquam, ipsum vel laoreet gravida, nunc mauris tempus lorem, sed iaculis odio libero vitae lectus. In nec euismod metus. In vel elit semper, sagittis nulla sed, aliquet est. Proin finibus risus sem, fermentum eleifend eros volutpat a.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.